contracts/libraries/AddressConverters.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+library Bytes32ToAddress {
+    function toAddress(bytes32 _bytes32) internal pure returns (address) {
+        return address(uint160(uint256(_bytes32)));
+    }
+}
+
+library AddressToBytes32 {
+    function toBytes32(address _address) internal pure returns (bytes32) {
```

Comment:
I like this. we should re-use it here as well: https://github.com/across-protocol/contracts/blob/5f08c27dd09f8c674ccc4af39d8b6b7c87a0444e/contracts/libraries/CircleCCTPAdapter.sol#L71